app/src/main/java/com/toshi/manager/chat/tasks/GroupUpdateTask.kt

```
@@ -50,10 +50,11 @@ class GroupUpdateTask(
         val signalGroup = dataMessage.groupInfo.get()
         Group
                 .fromSignalGroup(signalGroup)
+                .flatMapCompletable { messageSender.sendGroupInfo(messageSource, it) }
                 .subscribeOn(Schedulers.io())
                 .observeOn(Schedulers.io())
                 .subscribe(
-                        { group -> messageSender.sendGroupInfo(messageSource, group) },
+                        {},
```

Why this change? Haven't an empty subscription just looks like a mistake.

---

I made the `MessageSender:: sendGroupInfo` a completable

---

Why?

---

Because `SendGroupInfoTask::run` is a completable. I basically made every tasks that throws `EncapsulatedExceptions` in a completable or single to ignore the exception when users have signed out. It makes chaining things together easier